spec/features/navigation_spec.rb

```diff
@@ -59,6 +59,24 @@
 
         expect(current_path).to eq("/register")
       end
+      
+      it "can redirect an admin to 404 error if they try to access any path with /merchant" do
```

Comment:
This was an intense git merge! Glad we worked through that together and learned that we need to do separate tests or setups for merges